README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TODO LIST
 
-> One paragraph statement about the project.
+> "TODO List is simple task organizer written in javascript.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
```

Comment:
- [ ] Kindly delete `node_modules` directory from your repository and make sure you add it to `gitignore` file as it not necessary and it makes your repository to be crowded.